Equilobe.DailyReport.SL/JiraService.cs

```diff
@@ -87,6 +88,22 @@ public List<JiraUser> GetAllUsers(JiraRequestContext context)
             return GetClient(context).GetAllUsers();
         }
 
+        public List<JiraIssue> GetWorklogsForMultipleUsers(JiraRequestContext context, List<string> authors, DateTime fromDate, DateTime toDate)
+        {
+            var startDate = fromDate.ToString("yyyy-MM-dd");
```

Comment:
Make sure to convert the dates (throughout the app, where is the case) to the original timezone. There is already an extension method for that. Otherwise, some logs may not be included in the queries.